llama_stack/distribution/routers/routing_tables.py

```diff
@@ -461,3 +465,61 @@ async def register_eval_task(
             provider_resource_id=provider_eval_task_id,
         )
         await self.register_object(eval_task)
+
+
+class ToolsRoutingTable(CommonRoutingTableImpl, Tools):
+    async def list_tools(self) -> List[Tool]:
+        return await self.get_all_with_type("tool")
+
+    async def get_tool(self, tool_id: str) -> Tool:
+        return await self.get_object_by_identifier("tool", tool_id)
+
+    async def register_tool_group(
+        self,
+        name: str,
+        tool_group: ToolGroup,
+        provider_id: Optional[str] = None,
+    ) -> None:
+        tools = []
+        if isinstance(tool_group, MCPToolGroup):
+            # TODO: first needs to be resolved to corresponding tools available in the MCP server
```

Comment:
btw it would be good to find a simple publicly available MCP server if there is one. maybe github offers, or maybe anthropic itself offers something.

Reply:
yup, will look in to this.